crates/ruff_linter/src/rules/flake8_raise/rules/unnecessary_paren_on_raise_exception.rs

```diff
@@ -15,6 +16,13 @@ use crate::checkers::ast::Checker;
 ///
 /// Removing the parentheses makes the code more concise.
 ///
+/// ## Known problems
+/// Parentheses can only be omitted if the exception is a class, as opposed to
+/// a function call. This rule isn't always capable of distinguishing between
+/// the two. For example, if you define a method `get_exception` that itself
+/// returns an exception object, this rule will falsy mark the parentheses
```

Comment:
```suggestion
/// returns an exception object, this rule will incorrectly mark the parentheses
```

Reply:
I'm also a little confused as I thought function definitions were excluded?

Reply:
Improved in https://github.com/astral-sh/ruff/pull/8256, thank you!